src/ui/components/VPNAlerts.qml

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.5
+import Mozilla.VPN 1.0
+import QtQuick.Layouts 1.14
+import "./../components"
+import "./../themes/themes.js" as Theme
+
+ColumnLayout {
+    id: notifications
+    spacing: Theme.windowMargin / 2
+    Layout.maximumWidth: col.width - Theme.windowMargin
+    Layout.alignment: Qt.AlignHCenter
+    Layout.fillHeight: false
+    visible: VPNSurveyModel.hasSurvey || updateAlert.visible
+
+    VPNAlert {
+        id: updateAlert
+
+        state: VPN.updateRecommended ? "recommended" : ""
+        alertType: "update"
+        alertColor: Theme.blueButton
+        visible: state === "recommended"
+        //% "New version is available."
+        alertText: qsTrId("vpn.updates.newVersionAvailable")
+        //% "Update now"
+        alertLinkText: qsTrId("vpn.updates.updateNow")
+        isLayout: true
+    }
+
+    VPNAlert {
+        id: surveyAlert
+
+        isLayout: true
+        alertType: "survey"
+        //% "TBD"
```

Comment:
Please don't do this? There's a pipeline attached to this project, and now I have "TBD" strings queued for merge and translation?
https://github.com/mozilla-l10n/mozilla-vpn-client-l10n/pull/133/files